<ui version="4.0" >
 <class>shglyphsgui</class>
 <widget class="QMainWindow" name="shglyphsgui" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QHBoxLayout" >
    <item>
     <widget class="QVTKWidget" native="1" name="vtkWidget" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>200</width>
        <height>16</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Seeding</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Distance:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="seedDistanceSpinBox" >
              <property name="minimum" >
               <double>0.100000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Appearance</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Scaling:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="scalingSpinBox" >
              <property name="minimum" >
               <double>0.010000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="localScalingCheckBox" >
            <property name="text" >
             <string>Local scaling</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Sharpening:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="sharpeningSpinBox" >
              <property name="decimals" >
               <number>1</number>
              </property>
              <property name="minimum" >
               <double>0.100000000000000</double>
              </property>
              <property name="maximum" >
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Ray casting</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Step size:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="stepSizeSpinBox" >
              <property name="decimals" >
               <number>3</number>
              </property>
              <property name="minimum" >
               <double>0.000000000000000</double>
              </property>
              <property name="maximum" >
               <double>100.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.010000000000000</double>
              </property>
              <property name="value" >
               <double>0.010000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>#Refinement steps:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QSpinBox" name="refinementSpinBox" >
              <property name="maximum" >
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="refinementSlider" >
              <property name="maximum" >
               <number>100</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>Testing</string>
         </property>
         <layout class="QVBoxLayout" >
          <item>
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Z rotation angle:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QSpinBox" name="zRotationSpinBox" >
              <property name="maximum" >
               <number>360</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="zRotationSlider" >
              <property name="maximum" >
               <number>360</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>Y rotation angle:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QSpinBox" name="yRotationSpinBox" >
              <property name="maximum" >
               <number>360</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="yRotationSlider" >
              <property name="maximum" >
               <number>360</number>
              </property>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>178</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>refinementSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>refinementSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>741</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel" >
     <x>844</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refinementSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>refinementSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>844</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel" >
     <x>741</x>
     <y>316</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zRotationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zRotationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>775</x>
     <y>399</y>
    </hint>
    <hint type="destinationlabel" >
     <x>717</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotationSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yRotationSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>724</x>
     <y>466</y>
    </hint>
    <hint type="destinationlabel" >
     <x>752</x>
     <y>465</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotationSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yRotationSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>831</x>
     <y>468</y>
    </hint>
    <hint type="destinationlabel" >
     <x>713</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
